beta-src/src/components/ui/WDMoveControls.tsx

```diff
@@ -7,7 +7,7 @@ import useViewport from "../../hooks/useViewport";
 import getDevice from "../../utils/getDevice";
 import Device from "../../enums/Device";
 import { useAppDispatch, useAppSelector } from "../../state/hooks";
-import {
+import gameApiSlice, {
```

Comment:
your not using gameApiSlice import in this file

Reply:
Are you using this import? Why is this conversation resolved?